src/components/ExpandableSection/ExpandableSection.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import { Button, ButtonIcon } from '../../';
+
+class ExpandableSection extends Component {
+  constructor(props) {
+    super(props);
+    const { open, collapsable } = props;
+
+    if (props.defaultOpen != null) {
+      if (!collapsable) this.state = { open: true };
+      const defaultOpen = this.props.defaultOpen;
+      this.state = { open: defaultOpen };
+    }
+
+    if (props.defaultOpen == null) {
+      if (open != null) {
+        if (open === false && !collapsable) {
+          // eslint-disable-next-line
+          console.warn(
+            '[react-lds] ExpandableSection:',
+            'When collapsable is false, open cannot be false.',
+          );
+        }
+        this.state = { open };
+      } else {
+        // eslint-disable-next-line
+        console.warn(
+          '[react-lds] ExpandableSection:',
+          'Either defaultOpen or open need to be != null.',
+        );
+      }
+    }
+  }
+
+  onClickToggle = (nextOpenState) => {
+    this.setState(() => ({ open: nextOpenState }));
```

Comment:
If this only toggles with a click, you can simplify this to:

``` js
onClickToggle = () => {
  this.setState(prevState => { open: !prevState.open });
}
```

The added benefit is safety in case you update from outside (react updates are batched)